src/main/java/it/cnr/isti/hpc/wikipedia/parser/ArticleParser.java

```diff
@@ -157,7 +159,7 @@ private void setWikiTitle(Article article) {
 	 */
 	private void setIsList(Article article) {
 		for (String list : locale.getListIdentifiers()) {
-			if (StringUtils.startsWithIgnoreCase(article.getTitle(), list)) {
+			if (StringUtils.startsWithIgnoreCase(article.getTitle(), list + " ")) {
```

Comment:
If we do an upstream fix, then we can think about multi-lingual stuff. For now I just want to do this for English.